prisma/schema.prisma

```diff
@@ -86,7 +86,7 @@ model Puzzle {
   question        String
   published       Boolean      @default(false)
   variables       Json
-  type            PuzzleType   @relation(fields: [puzzleTypeId], references: [id])
+  puzzleType      PuzzleType   @relation(fields: [puzzleTypeId], references: [id])
```

Comment:
I think you also need a `@map` for this

Reply:
good catch!

Reply:
Oh actually my editor said that `@map` can't be used on `@relation` fields, I think it's because technically the `puzzleTypeId` is the thing that's actually stored in the database